lib/active_model/serializer/collection_serializer.rb

```diff
@@ -52,7 +52,8 @@ def json_key
       # rubocop:enable Metrics/CyclomaticComplexity
 
       def paginated?
-        object.respond_to?(:current_page) &&
+        Serializer.config.jsonapi_pagination_links_enabled &&
```

Comment:
`s/Serializer/ActiveModelSerializers`. the Serializer is no longer the primary object..